src/test/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/RepaymentPlanContentProviderTest.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.cmc.claimstore.services.staff.content;
+
+import org.junit.Test;
+import uk.gov.hmcts.cmc.claimstore.services.staff.models.RepaymentPlanContent;
+import uk.gov.hmcts.cmc.domain.models.RepaymentPlan;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleRepaymentPlan;
+
+import java.math.BigDecimal;
+
+import static java.time.LocalDate.now;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.cmc.claimstore.services.staff.content.RepaymentPlanContentProvider.create;
+import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatDate;
+import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatMoney;
+import static uk.gov.hmcts.cmc.domain.models.PaymentOption.FULL_BY_SPECIFIED_DATE;
+import static uk.gov.hmcts.cmc.domain.models.PaymentOption.IMMEDIATELY;
+import static uk.gov.hmcts.cmc.domain.models.PaymentOption.INSTALMENTS;
+
+public class RepaymentPlanContentProviderTest {
+    @Test
+    public void createImmediatePaymentOption() {
+
```

Comment:
remove first blank line from each of these test methods please, it gives no value